### PR TITLE
catch more instances where add_arguments is needed

### PR DESCRIPTION
**Pull request by nickpell**
@emord cc: @mkangia

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/upload_app_forms.py [hunk 1]
```diff
@@ -45,9 +45,7 @@ def add_arguments(self, parser):
         )
 
 
-    def handle(self, *args, **options):
-        if len(args) != 2:
-            raise CommandError('Usage: %s\n%s' % (self.args, self.help))
+    def handle(self, path, app_id, **options):
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/upload_app_forms.py]: E303 too many blank lines (2)

File: corehq/ex-submodules/couchforms/management/commands/purge_forms_and_cases.py [hunk 2]
```diff
@@ -155,6 +159,6 @@ def _print_final_debug_info(xform_ids, filtered_xform_ids, case_ids):
 
 
 def _notify_parsed_args(domain_name, app_id, version_number, test_run):
-    print('Received request for domain : %s with app_id : %s to soft delete data before version number %s' % \
+    print('Received request for domain : %s with app_id : %s to soft delete data before version number %d' % \
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/management/commands/purge_forms_and_cases.py]: E502 the backslash is redundant between brackets

**Comment by emord** [comment]
DIdn't look too hard, but the general idea seems good.

However part of me wants to say don't commit this. Instead you could add a comment or some message to each of these saying "Whoa this actually gets run? Here's a link to a commit that will fix it" And then after six months we could look through these and see which were changed and delete the ones that weren't.

(also you've already done this work so feel free to go ahead and merge)

**Comment by nickpell** [comment]
interesting idea.  What about putting at the start of every infrequently used management command
```raise Exception("Delete this command if not used by 2017-10-01")```
or the like?

**Comment by emord** [comment]
Yeah.that seems good too

**Pull request merged**